tests/conditional_compilation/test_size_tracking.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from path_utils import library_name  # pylint: disable=import-error
+
+
+def get_lib_sizes(path, libraries):
+    """Function for getting lib sizes by lib names"""
+    assert Path.exists(path), f'Directory {path} isn\'t created'
+    result = {}
+    error_lib = []
+    for lib in libraries:
+        try:
+            result[lib] = Path(path).joinpath(library_name(lib)).stat().st_size
+        except FileNotFoundError:
+            error_lib.append(lib)
+    assert len(error_lib) == 0, 'Following libraries couldn\'t be found: {}'.format(error_lib)
+    return result
+
+
+def test_size_tracking_libs(openvino_root_dir, test_id, model, artifacts):
+    libraries = ['inference_engine_transformations', 'MKLDNNPlugin', 'ngraph']
+
+    ref_libs_size = get_lib_sizes(openvino_root_dir, libraries)
+    install_prefix = artifacts / test_id / 'install_pkg'
+    lib_sizes = get_lib_sizes(install_prefix, libraries)
+
+    for lib in libraries:
+        lib_size_diff = ref_libs_size[lib] - lib_sizes[lib]
+        lib_delta = lib_size_diff / ref_libs_size[lib] * 100
+        print('{}: old - {}kB; new - {}kB; diff = {}kB({:.2f}%)'.format(lib,
+                                                                        ref_libs_size[lib] / 1024,
+                                                                        lib_sizes[lib] / 1024,
+                                                                        lib_size_diff / 1024,
+                                                                        lib_delta))
+    res = [key for key in lib_sizes.keys() if lib_sizes[key] > ref_libs_size[key]]
```

Comment:
I see no guarantee that `lib_sizes.keys()` == `ref_libs_size.keys()`. Should we compare only libs defined in `libraries`, or variables should contain equal keys? If compare only from `libraries`, need to replace `for key in lib_sizes.keys()` with `for key in libraries`

Reply:
Variables contain equal keys. They are initialized by one function but with different installation directories.

Reply:
OK. Anyway, I would prefer to iterate from `libraries` here because it's a guarantee of equal keys, and in case of some changes it will be more robust